### PR TITLE
Merge master-vs-deps to master

### PR DESCRIPTION
**Pull request by brettfo**

### CODE REVIEW

**Comment by sharwell** [comment]
test vsi please

**Comment by sharwell** [comment]
test windows_debug_vs-integration_prtest please

**Comment by jasonmalinowski** [comment]
@dotnet-bot retest this please.

**Comment by jasonmalinowski** [comment]
@dotnet-bot retest windows_debug_vs-integration_prtest please.

**Comment by jasonmalinowski** [comment]
@dotnet-bot retest windows_release_vs-integration_prtest please.

**Comment by jasonmalinowski** [comment]
@sharwell Are these errors something I should be investigating or no?

**Comment by jasonmalinowski** [comment]
And oops, resolved a conflict and reran the tests. :frowning:

**Comment by jasonmalinowski** [comment]
@dotnet-bot retest this please.

**Comment by dpoeschl** [comment]
`windows_debug_vs-integration_prtest` --

**Comment by dpoeschl** [comment]
@dotnet-bot help

**Comment by dpoeschl** [comment]
retest windows_release_vs-integration_prtest please

**Comment by dpoeschl** [comment]
retest windows_debug_vs-integration_prtest please

**Comment by dpoeschl** [comment]
And for some reason that triggered them all.

File: src/NuGet/VisualStudio/VS.ExternalAPIs.Roslyn.Package.csproj [hunk 1]
```diff
@@ -44,6 +44,7 @@
     <ProjectReference Include="..\..\VisualStudio\Core\Def\Microsoft.VisualStudio.LanguageServices.csproj" PrivateAssets="all"/>
     <ProjectReference Include="..\..\VisualStudio\Core\Impl\Microsoft.VisualStudio.LanguageServices.Implementation.csproj" PrivateAssets="all"/>
     <ProjectReference Include="..\..\VisualStudio\Core\SolutionExplorerShim\Microsoft.VisualStudio.LanguageServices.SolutionExplorer.csproj" PrivateAssets="all"/>
+    <ProjectReference Include="..\..\VisualStudio\Core\SolutionExplorerShim\Microsoft.VisualStudio.LanguageServices.SolutionExplorer.csproj" PrivateAssets="all"/>
```
Comment (sharwell) [inline src/NuGet/VisualStudio/VS.ExternalAPIs.Roslyn.Package.csproj]: @jasonmalinowski Intentional duplicate reference?

Comment (jasonmalinowski) [reply src/NuGet/VisualStudio/VS.ExternalAPIs.Roslyn.Package.csproj]: @sharwell No, that wasn't. Did you fix this one up?

Comment (sharwell) [reply src/NuGet/VisualStudio/VS.ExternalAPIs.Roslyn.Package.csproj]: > No, that wasn't. Did you fix this one up?

I did not. Once I had a passing build I didn't want to chance the build queue ignoring my manual override and starting a new cycle of failures.

**Pull request merged**